dot/rpc/subscription/messages.go

```diff
@@ -59,3 +65,18 @@ func newSubscriptionResponseJSON(subID uint, reqID float64) ResponseJSON {
 		ID:      reqID,
 	}
 }
+
+// BooleanResponseJSON for responses that return boolean values
+type BooleanResponseJSON struct {
+	Jsonrpc string  `json:"jsonrpc"`
+	Result  bool    `json:"result"`
+	ID      float64 `json:"id"`
+}
+
+func newBooleanResponseJSON(value bool, reqID float64) BooleanResponseJSON {
```

Comment:
if the type is public why not the constructor?

Reply:
No reason... I've updated this.